### PR TITLE
Updated JSON config for motion test.

### PR DESCRIPTION
**Pull request by SkyToGround**
As the title says.

## Issue(s)

See JIRA issues:

- ECDC-2136
- ECDC-2362

### CODE REVIEW

File: nicos_ess/ymir/commands/nexus_config.json [hunk 1]
```diff
@@ -1,97 +1,231 @@
...
             }
-          ]
+          ],
+          "children": []
         }
       ]
     }
   ]
-}
+}
```
Comment (SkyToGround) [inline nicos_ess/ymir/commands/nexus_config.json:231]:
```suggestion
}

```

**Comment by umithardal** [comment]
@SkyToGround Are these changes require test before merging or should be merged so that they can be tested?

**Review by umithardal — APPROVED** [review]
LGTM.

**Review by kmurica — APPROVED** [review]
LGTM

**Pull request merged**